KTLOSBWLLoot.json

```diff
@@ -603,7 +603,7 @@
 			"wowID":""
 		},
 		"Ashkandi Greatsword of the Brotherhood":{
-			"prio":["Chrixus(1)","Oph(1)","PVP prio","Warriors, Hunter, Pally","EP/GP"],
+			"prio":["Chrixus(2)","Oph(1)","PVP prio","LC","Warriors, Hunter, Pally","EP/GP"],
```

Comment:
This was also a last minute change by one James, was this discussed?  If not we should roll this back

Reply:
we discussed it, we're going to keep Oph in